### PR TITLE
[18858] Bump version to v1.1.0

### LINKED ISSUE #103: Add installation instructions to Readme [13750]
It took me awhile to find this page. I think you should put the link in the README:  [eprosima installation](https://fast-dds.docs.eprosima.com/en/latest/installation/binaries/binaries_linux.html)

### PR DESCRIPTION
**Pull request by EduPonz**
This PR sets the project version to v1.1.0. Additionally:

* Update the platform support
* Closes #103

### CODE REVIEW

File: PLATFORM_SUPPORT.md [hunk 1]
```diff
@@ -31,12 +31,12 @@ Community members may provide assistance with these platforms.
 
 ## Platforms
 
-|Architecture|Ubuntu Focal (20.04)|MacOS Mojave (10.14)|Windows 10 (VS2019)|Debian Buster (10)|
-|------------|--------------------|--------------------|-------------------|------------------|
-|amd64       |Tier 1 [a][s]       |Tier 1 [s]          |Tier 1 [a][s]      |Tier 3 [s]        |
-|amd32       |                    |                    |Tier 1 [a][s]      |                  |
-|arm64       |Tier 1 [a][s]       |                    |                   |Tier 3 [s]        |
-|arm32       |Tier 3 [s]          |                    |                   |Tier 3 [s]        |
+|Architecture|Ubuntu Focal (22.04)|MacOS Mojave (10.14)|Windows 10 (VS2019)|Ubuntu Focal (20.04)|Debian Buster (10)|
```
Comment (jsantiago-eProsima) [inline PLATFORM_SUPPORT.md:34]:
```suggestion
|Architecture|Ubuntu Jammy (22.04)|MacOS Mojave (10.14)|Windows 10 (VS2019)|Ubuntu Focal (20.04)|Debian Buster (10)|
```

File: PLATFORM_SUPPORT.md [hunk 2]
```diff
@@ -45,4 +45,4 @@ Other Tier 3 OS:
 
 * FreeBSD
 * VxWorks
-* QNX
\ No newline at end of file
+* QNX
```
Comment (jsantiago-eProsima) [inline PLATFORM_SUPPORT.md:48]: Since FastDDS is currently supporting Android as Tier3, should it be added as well?

File: README.md [hunk 3]
```diff
@@ -16,8 +16,8 @@ One is the standard CDR serialization mechanism, while the other is a faster imp
 
 ## Build
 
-**eProsima Fast CDR** provides [CMake][cmake] scripts to build and install it.
-Also in [eProsima][eprosima] you can find packages for Linux using autotools and binaries for Windows.
+**eProsima Fast CDR** provides [CMake][cmake] scripts to build and install it (please read the [installation guide](https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources_linux.html#) for more details).
```
Comment (jsantiago-eProsima) [inline README.md:19]: Not really clear which part of the Fast DDS installation a user should be looking at. The CMake section? Maybe we should link that one directly.

**Review by jsantiago-eProsima — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -16,8 +16,8 @@ One is the standard CDR serialization mechanism, while the other is a faster imp
 
 ## Build
 
-**eProsima Fast CDR** provides [CMake][cmake] scripts to build and install it.
-Also in [eProsima][eprosima] you can find packages for Linux using autotools and binaries for Windows.
+**eProsima Fast CDR** provides [CMake][cmake] scripts to build and install it (please read the [installation guide](https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources_linux.html#cmake-installation-python-linux) for more details).
```
Comment (jsantiago-eProsima) [inline README.md:19]: Small nitpick:

```suggestion
**eProsima Fast CDR** provides [CMake][cmake] scripts to build and install it (please read the [installation guide](https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources_linux.html#cmake-installation) for more details).
```

**Review by jsantiago-eProsima — CHANGES_REQUESTED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Linux/190/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Mac/116/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Windows/182/)

**Review by jsantiago-eProsima — APPROVED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Linux/200/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Mac/126/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Windows/192/)

**Pull request merged**